### ISSUE TITLE
Introducing "new" tag for new activity cards

### ISSUE THREAD

**Issue by sssash18** [publiclab/plots2#7353]
**Please describe the desired behavior.**
Describe the solution you'd like.
It will be good if we put a new tag in user's activity which are added after the user's last login.
**Current Behaviour**
![Screenshot 2020-01-26 at 4 59 50 PM](https://user-images.githubusercontent.com/57896905/73134525-5925b100-405d-11ea-9133-1b084c9d0577.png)

**Expected Behaviour**
![Screenshot 2020-01-26 at 4 54 10 PM](https://user-images.githubusercontent.com/57896905/73134520-4d39ef00-405d-11ea-8c15-a0cbcfc2d579.png)

**Additional context (optional)**
Feel free to include any applicable: code, screenshots, or gifs.

**Comment by sssash18** [comment]
@publiclab/reviewers  @SidharthBansal  What are your views on this?
I would like to implement this.

**Comment by sssash18** [comment]
@SidharthBansal @jywarren Please share your views on this?I would like to do it.

**Comment by nstjean** [comment]
That looks pretty neat. How would you calculate which to show as "new"?

**Comment by sssash18** [comment]
@nstjean I will compare the time of last login and the time of post created.
if time of post created > time of last login "new"  tag will be shown.

**Comment by nstjean** [comment]
Sounds great. I like it. But I'll let the others give an opinion. :)

**Comment by sssash18** [comment]
@nstjean Thanks for your approval. Let's wait for other members views.

**Comment by SidharthBansal** [comment]
I am in favour of it. Let's ask @jywarren 
What is github username for Emily?

**Comment by sssash18** [comment]
> I am in favour of it. Let's ask @jywarren
> What is github username for Emily?

"@emilyashley" is the username.

**Comment by SidharthBansal** [comment]
Thanks for pinging Emily

**Comment by sssash18** [comment]
@SidharthBansal any update on this?

**Comment by sssash18** [comment]
@SidharthBansal @nstjean Any update on this?

**Comment by SidharthBansal** [comment]
I am in favour of it. Pinging @jywarren

**Comment by jywarren** [comment]
Hi, I think this could work but let's aim to make it compact, maintainable, and easy to read. We could create: 

* [ ] a `Node` model method called `node.is_new_to_user(user)` with a good comment explaining the metric
* [ ] a unit test confirming this in positive and negative cases
* [ ] a post template addition adding the `new` badge (this could be a little smaller maybe?)

How does this sound? Thank you!

**Comment by sssash18** [comment]
Yes @jywarren It seems nice! I will try to implement as suggested by you.Thanks!

**Comment by gauravano** [comment]
Hi @sssash18, I liked the idea. 

We can also check for how long a session persists on website. I last logged into the site say 2 weeks ago or maybe more, but I just opened the website and I am still logged in. So, we should find those triggers which help us show the tag correctly.   

The above idea can be in V2 too.

Thanks!
